.github/workflows/reproducible-builds.yml

```diff
@@ -111,10 +110,11 @@ jobs:
         run: shasum -a 256 "${binary}" | tee "${binary}.sha256"
 
       - name: Upload binary artifact
-        uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
+        uses: ./.github/actions/artifact_upload
         with:
           name: "osimages-${{ matrix.target }}"
           path: "${{ env.binary }}"
+          encryptionSecret: ${{ secrets.ARTIFACT_ENCRYPT_PASSWD }}
```

Comment:
Is there a reason for encrypting here? It seems that a binary is uploaded, right?